translations/ui-oai-pmh/en.json

```diff
@@ -83,6 +83,8 @@
   "settings.sets.callout.connectionProblem.get": "Unable to get data due to a module error. Please try again. If a problem persists please contact your system administrator",
   "settings.sets.callout.connectionProblem": "Unable to save due to a module error. Please try again. If a problem persists please contact your system administrator",
 
+  "settings.sets.callout.validationError.notUnique": "Set name or setSpec isn't unique. Please, fill in unique set name or setSpec to save set.",
+
```

Comment:
I realize this message was listed in https://issues.folio.org/browse/UIOAIPMH-18 and this may be more of a requirements question -- but I find the terminology a little confusing - when I view http://www.openarchives.org/OAI/2.0/openarchivesprotocol.htm#Set it mentions setName and setSpec -- but when I look at the UI elements are labeled as `set name` and `specification` - should message be more reflective of UI element naming

Reply:
This is about another field that to be added in scope of https://issues.folio.org/browse/UIOAIPMH-21. I will discuss text for current message with PO.

Reply:
Fixed.